pandas/tests/scalar/timedelta/test_arithmetic.py

```diff
@@ -412,6 +412,46 @@ def test_td_rdiv_timedeltalike_scalar(self):
 
         assert np.timedelta64(60, "h") / td == 0.25
 
+    def test_td_rdiv_na_scalar(self):
+        # GH#31869 None gets cast to NaT
+        td = Timedelta(10, unit="d")
+
+        result = NaT / td
```

Comment:
maybe as a follown split out the these non-error casees and use nulls_fixture (as you then can test float('nan')